citing-us/index.html

```diff
@@ -0,0 +1,88 @@
+---
+layout: default
+title: Citing OME
+---          
+        <div class="callout large primary" id="bg-image-licensing">
+            <div class="row column text-center">
+                <h1>Citing Us</h1>
+                <p class="hero-subheader small-12 medium-12">We are delighted to have you cite the work of the Open Microscopy Environment from your own publications.</p>
```

Comment:
"Please cite our work in your own publications to help us secure grant funding for future development of OME products." ?